blocks/library/freeform/test/naiveCss2Jsx.js

```diff
@@ -1,10 +1,6 @@
 /**
  * External dependencies
  */
-import { expect } from 'chai';
-/**
- * Internal dependencies
- */
 import { naiveCss2Jsx } from '../format-list';
```

Comment:
This section should now be titled `Internal dependencies`, not `External dependencies`.